typings/single-spa.d.ts

```diff
@@ -72,3 +93,7 @@ declare module "single-spa" {
   // './parcels/mount-parcel.js'
   export function mountRootParcel(parcelConfig: any, parcelProps: any): boolean;
 }
+
+declare module 'single-spa' {
+  export = SingleSpa;
```

Comment:
I'm not an expert in Typescript, but this change worries me. Could you explain to me the implications of doing `export = SingleSpa` instead of exporting each of the functions individually? If we were to introduce a `default` export in the future to single-spa, would it mess up the type definitions since `export =` is cjs/esm compatibility and cjs doesn't have the concept of default exports?

Could be totally wrong here, but just would like to understand these changes better. Am basing what I said off of https://www.typescriptlang.org/docs/handbook/modules.html